params/superchain.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
-var OPStackSupport = ProtocolVersionV0{Build: [8]byte{}, Major: 3, Minor: 1, Patch: 0, PreRelease: 0}.Encode()
+var OPStackSupport = ProtocolVersionV0{Build: [8]byte{}, Major: 3, Minor: 1, Patch: 0, PreRelease: 1}.Encode()
```

Comment:
This now _decreases_ compatibility, I think we wanted to set this to 3.1.0 so this node can already run on mainnet, once a recent release is finalized. cc @protolambda

Reply:
Oh, should it be 3.2.0 prerelease 1? Sorry, I'm not super familiar with these either

Reply:
Or should I just leave this as is?

Reply:
FYI this is the first version compatible with canyon, so maybe we do want to bump it a version.

Reply:
Canyon can be considered a major breaking change in terms of semver, but is not finalized yet. It should be set to `v4.0.0-1`, to signal readiness for a pre-release of the next major upgrade.

Reply:
oh yeah, `4.0.0-1`

Reply:
will bump to `4.0.0` when we're cutting the mainnet release.